web/themes/custom/votegov/templates/navigation/menu--main--votegov-mainnavigation.html.twig

```diff
@@ -35,6 +35,10 @@ We call a macro which calls itself to render the full tree.
   {{ menus.menu_links(items, attributes, 0, 'basic-mega-nav-section-') }}
   <div class="usa-nav__secondary">
     {{ drupal_entity('block_content', 2) }}
+     {% if language == 'en' %}
+        <button class="usa-button" id="vote-touchpoints-mobile--button" title="{{ 'How can we improve vote.gov?' | t }}" data-test="touchpointsBtn"
+         onclick="document.getElementById('vote-touchpoints--button').click()">{{ 'How can we improve vote.gov?' | t }}</button>
```

Comment:
It is not recommended to have onclick functions in the dom. Can you move this to a js file and load it with the button?